examples/08-averaging/01-average_across_bodies.py

```diff
@@ -0,0 +1,283 @@
+# As we can see in the image above, even though the piston rod is one single part,
+# it's composed of two different bodies. Additionally, we can observe that the region
+# where the two bodies are bonded together contains nodes that are common between them.
+
+###############################################################################
+# Now, let's take a look into how the averaging across bodies option alters the
+# results of a simulation.
+
+###############################################################################
+# Averaging across bodies with DPF
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Let's define two workflows. The first one does averaging across bodies, while the
+# second one doesn't. The variable of interest here is the stress in the Z direction,
+# which will be obtained using the "stress_Z" operator.
+
+# %%
+# .. graphviz::
+#
+#   digraph foo {
+#       graph [pad="0", nodesep="0.3", ranksep="0.3"]
+#       node [shape=box, style=filled, fillcolor="#ffcc0", margin="0"];
+#       rankdir=LR;
+#       splines=line;
+#       node [fixedsize=true,width=2.5]
+#       stress01 [label="stress_Z"];
+#       stress02 [label="stress_Z"];
+#       scp01 [label="split_on_property_type"];
+#       subgraph cluster_1 {
+#           ds01 [label="data_src", shape=box, style=filled, fillcolor=cadetblue2];
+#           ds01 -> scp01 [style=dashed];
+#           scp01 -> stress01;
+#           label="Averaging across bodies = Off";
+#           style=filled;
+#           fillcolor=lightgrey;
+#       }
+#       subgraph cluster_2 {
+#           ds02 [label="data_src", shape=box, style=filled, fillcolor=cadetblue2];
+#           inv02 [style=invisible]
+#           ds02 -> stress02 [style=dashed];
+#           stress02 -> inv02 [style=invis]
+#           label="Averaging across bodies = On";
+#           style=filled;
+#           fillcolor=lightgrey;
+#       }
+#   }
+
+
+###############################################################################
+# Averaging across bodies activated
+# ---------------------------------
+# The extraction of the stresses in the Z direction in DPF applies by default averaging
+# across bodies. Therefore, a simple workflow like the one shown below can be used
+# in this case.
+
+
+def average_across_bodies(analysis):
+    # This function will extract the stresses in the Z direction (with the average
+    # across bodies property activated) and plot them.
+
+    # Create a model from the simulation results.
+    model = dpf.Model(analysis)
+    mesh = model.metadata.meshed_region
+
+    # We're interested in the last time step, so:
+    time_step = 3
+
+    # Extracting the stresses in the Z direction. By default, DPF already applies
+    # averaging across bodies when extracting the stresses.
+    stress_op = ops.result.stress_Z()
+    stress_op.inputs.connect(model)
+    stress_op.inputs.time_scoping.connect(time_step)
+    stress_op.inputs.requested_location.connect("Nodal")
```

Comment:
could you please use:
```suggestion
    stress_op.inputs.requested_location.connect(dpf.locations.nodal)
```